assets/js/blocks/cart/cart-cross-sells-product-list/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { ProductResponseItem } from '@woocommerce/type-defs/product-response';
+
+/**
+ * Internal dependencies
+ */
+import CartCrossSellsProduct from './cart-cross-sells-product';
+
+const placeholderRows = [ ...Array( 3 ) ].map( ( _x, i ) => (
+	<CartCrossSellsProduct crossSellsProduct={ {} } key={ i } />
```

Comment:
This doesn't seem to do anything, I replaced it with placeholder images and they are never rendered. Also, I'm curious if we considered creating a separate component called `CrossSellsProductPlaceholder` or something instead of a `CartCrossSellsProduct` with no data?

I see `CrossSellsProduct` has an `isLoading` prop, but this seems to be unused?

Reply:
I removed both `isLoading` and `placeholderRows`, given that the Cross-Sells product section will only appear if the products in the cart have Cross-Sells products.